src/inventory/gilded_rose.js

```diff
@@ -22,49 +22,77 @@ const items = [
 updateQuality(items);
 */
 export function updateQuality(items) {
+    //combine these into quality changer--takes in a + or - number and you can get BOTH increase or decrease in quality
+    qualityUpdater = (item, change_Amount) =>{
```

Comment:
It looks like you might be using a combination of camelcase (`myGreatVariable`) and snake case (`my_great_variable`). I'd use camelcase for consistency.